packages/financial-templates-lib/src/clients/ExpiringMultiPartyClient.js

```diff
@@ -24,7 +24,8 @@ class ExpiringMultiPartyClient {
     empAddress,
     collateralDecimals = 18,
     syntheticDecimals = 18,
-    priceFeedDecimals = 18
+    priceFeedDecimals = 18,
+    contractType = "ExpiringMultiParty" // Default to EMP for now to enable backwards compatibility with other bots. This will be removed as soon as the other bots have been updated to work with these contract types.
```

Comment:
this ensures that these changes will not break this client within the other bots that are yet to be changed.

Reply:
That's a reasonable default